qiita_pet/templates/sitebase.html

```diff
@@ -54,6 +54,14 @@
     <link rel="shortcut icon" href="{% raw qiita_config.portal_dir %}/static/img/favicon.ico">
 
     <script>
+      $(window).resize(function () {
+        $('body').css('padding-top', parseInt($('#navigation-bar').css("height"))+10);
```

Comment:
[From reading the jQuery docs](http://api.jquery.com/height/), I believe you want to use the `height` function directly i.e. `$('#navigation-bar').height()`, this way you don't have to worry about the units of this attribute, as it could return `500 px`, and you also don't need the `parseInt`.